client/src/Components/CarList.js

```diff
@@ -1,31 +1,29 @@
-import React from "react";
+import React  from "react";
 import { useQuery } from "@apollo/react-hooks";
-
+import Select from 'react-select';
 import { getUserDataQuery } from "../queries/queries";
 import LoadingSpinner from "./LoadingSpinner";
-
 function displayCars(data) {
   return data.cars.map(car => {
-    return (<li key={car.id}>
-      {car.displayName}({car.make} - {car.model})
-    </li>);
+      return { value: `${car.id}`, label: `${car.displayName} ${car.make} (${car.model})` }
   });
 }
-
-export default function() {
+export default function(props) {
   const { loading, error, data } = useQuery(getUserDataQuery);
   if (loading) {
     return <LoadingSpinner />;
   } else if (error) {
     debugger;
     return <p>Error! {error}</p>;
   } else {
-    debugger;
-
     return (
       <div>
-        <ul>{displayCars(data.userData)}</ul>
+        <Select 
+          value = {props.selectedOption}
```

Comment:
let's get eslint + prettier going.
airbnb code standards suggest no gap around the equals sign in JSX.

Reply:
I will get eslint and prettier set up tomorrow with the airbnb style guide and get phillip onto it as well